Occasional panic in release benchmarks

The bench release step in CI is sometimes failing in the past ~2 days, looking through the full logs shows a panic. I noticed it is happening for a few people:

https://github.com/denoland/deno/runs/8141440030?check_suite_focus=true
https://github.com/denoland/deno/runs/8132978773?check_suite_focus=true
https://github.com/denoland/deno/runs/8155825401?check_suite_focus=true

```
2022-09-02T12:36:18.3505406Z http_benchmark testing RUST hyper
2022-09-02T12:36:23.3500404Z /home/runner/work/deno/deno/target/release/test_server 4563
2022-09-02T12:36:23.3528614Z hyper hello
2022-09-02T12:36:28.3508025Z /home/runner/work/deno/deno/third_party/prebuilt/linux64/wrk -d 20s --latency http://127.0.0.1:4563/
2022-09-02T12:36:49.4547335Z Running 20s test @ http://127.0.0.1:4563/
2022-09-02T12:36:49.4547829Z   2 threads and 10 connections
2022-09-02T12:36:49.4548514Z   Thread Stats   Avg      Stdev     Max   +/- Stdev
2022-09-02T12:36:49.4550281Z     Latency   121.75us   86.47us   5.10ms   99.49%
2022-09-02T12:36:49.4551042Z     Req/Sec    41.93k     3.49k   96.15k    92.52%
2022-09-02T12:36:49.4551335Z   Latency Distribution
2022-09-02T12:36:49.4551600Z      50%  119.00us
2022-09-02T12:36:49.4551834Z      75%  122.00us
2022-09-02T12:36:49.4552079Z      90%  128.00us
2022-09-02T12:36:49.4552309Z      99%  143.00us
2022-09-02T12:36:49.4559063Z   1672233 requests in 20.10s, 140.34MB read
2022-09-02T12:36:49.4559465Z Requests/sec:  83196.83
2022-09-02T12:36:49.4559779Z Transfer/sec:      6.98MB
2022-09-02T12:36:49.4560290Z 
2022-09-02T12:36:50.0658792Z [0m[1m[31merror[0m: Uncaught Error: bad
2022-09-02T12:36:50.0659136Z   throw Error("bad");
2022-09-02T12:36:50.0659423Z [0m[31m        ^[0m
2022-09-02T12:36:50.0659950Z     at [0m[1m[3mfoo[0m ([0m[36mfile:///home/runner/work/deno/deno/cli/tests/testdata/error_001.ts[0m:[0m[33m2[0m:[0m[33m9[0m)
2022-09-02T12:36:50.0660550Z     at [0m[1m[3mbar[0m ([0m[36mfile:///home/runner/work/deno/deno/cli/tests/testdata/error_001.ts[0m:[0m[33m6[0m:[0m[33m3[0m)
2022-09-02T12:36:50.0661297Z     at [0m[36mfile:///home/runner/work/deno/deno/cli/tests/testdata/error_001.ts[0m:[0m[33m9[0m:[0m[33m1[0m
2022-09-02T12:36:53.8917223Z thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseFloatError { kind: Invalid }', test_util/src/lib.rs:2209:62
2022-09-02T12:36:53.8918163Z stack backtrace:
2022-09-02T12:36:53.9023567Z    0:     0x55987e602aad - std::backtrace_rs::backtrace::libunwind::trace::h8217d0a8f3fd2f41
2022-09-02T12:36:53.9024317Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/../../backtrace/src/backtrace/libunwind.rs:93:5
2022-09-02T12:36:53.9025095Z    1:     0x55987e602aad - std::backtrace_rs::backtrace::trace_unsynchronized::h308103876b3af410
2022-09-02T12:36:53.9025705Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
2022-09-02T12:36:53.9026431Z    2:     0x55987e602aad - std::sys_common::backtrace::_print_fmt::hc208018c6153605e
2022-09-02T12:36:53.9027016Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/sys_common/backtrace.rs:66:5
2022-09-02T12:36:53.9027855Z    3:     0x55987e602aad - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hf89a7ed694dfb585
2022-09-02T12:36:53.9028487Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/sys_common/backtrace.rs:45:22
2022-09-02T12:36:53.9040223Z    4:     0x55987e62a37c - core::fmt::write::h21038c1382fe4264
2022-09-02T12:36:53.9040737Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/fmt/mod.rs:1197:17
2022-09-02T12:36:53.9041406Z    5:     0x55987e5fc501 - std::io::Write::write_fmt::h7dbb1c9a3c254aef
2022-09-02T12:36:53.9041930Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/io/mod.rs:1672:15
2022-09-02T12:36:53.9042588Z    6:     0x55987e604385 - std::sys_common::backtrace::_print::h4e8889719c9ddeb8
2022-09-02T12:36:53.9043132Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/sys_common/backtrace.rs:48:5
2022-09-02T12:36:53.9043807Z    7:     0x55987e604385 - std::sys_common::backtrace::print::h1506fe2cb3022667
2022-09-02T12:36:53.9044350Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/sys_common/backtrace.rs:35:9
2022-09-02T12:36:53.9045036Z    8:     0x55987e604385 - std::panicking::default_hook::{{closure}}::hd9d7ce2a8a782440
2022-09-02T12:36:53.9045581Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:295:22
2022-09-02T12:36:53.9046239Z    9:     0x55987e6040a6 - std::panicking::default_hook::h5b16ec25444b1b5d
2022-09-02T12:36:53.9046765Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:314:9
2022-09-02T12:36:53.9047773Z   10:     0x55987e604976 - std::panicking::rust_panic_with_hook::hb0138cb6e6fea3e4
2022-09-02T12:36:53.9048318Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:698:17
2022-09-02T12:36:53.9049018Z   11:     0x55987e604867 - std::panicking::begin_panic_handler::{{closure}}::h4cb67095557cd1aa
2022-09-02T12:36:53.9049566Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:588:13
2022-09-02T12:36:53.9050291Z   12:     0x55987e602f64 - std::sys_common::backtrace::__rust_end_short_backtrace::h2bfcac279dcdc911
2022-09-02T12:36:53.9050873Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/sys_common/backtrace.rs:138:18
2022-09-02T12:36:53.9051395Z   13:     0x55987e604599 - rust_begin_unwind
2022-09-02T12:36:53.9051880Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:584:5
2022-09-02T12:36:53.9052558Z   14:     0x55987e628123 - core::panicking::panic_fmt::h1de71520faaa17d3
2022-09-02T12:36:53.9053090Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/panicking.rs:142:14
2022-09-02T12:36:53.9053843Z   15:     0x55987e6283d3 - core::result::unwrap_failed::hc0baa33ef8bc7db8
2022-09-02T12:36:53.9054367Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/result.rs:1805:5
2022-09-02T12:36:53.9055019Z   16:     0x55987e6eb61a - core::result::Result<T,E>::unwrap::h7e1240d879443ff1
2022-09-02T12:36:53.9055636Z   17:     0x55987e6f2426 - test_util::parse_strace_output::hebb5546f94939b80
2022-09-02T12:36:53.9056247Z   18:     0x55987e693da9 - deno_bench::main::{{closure}}::hef9d8fe059ae0e73
2022-09-02T12:36:53.9057019Z   19:     0x55987e67ab74 - <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::hbfe5c1a5749c31c1
2022-09-02T12:36:53.9057664Z   20:     0x55987e68cab7 - deno_bench::main::h2a6251fe232b87ef
2022-09-02T12:36:53.9058352Z   21:     0x55987e64c76f - std::sys_common::backtrace::__rust_begin_short_backtrace::h8922240d9524a44b
2022-09-02T12:36:53.9058997Z   22:     0x55987e64d395 - std::rt::lang_start::{{closure}}::h5f5c6fddb32353b1
2022-09-02T12:36:53.9059768Z   23:     0x55987e5f6d6e - core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once::h4937aaa125c8d4b2
2022-09-02T12:36:53.9060369Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/core/src/ops/function.rs:280:13
2022-09-02T12:36:53.9061015Z   24:     0x55987e5f6d6e - std::panicking::try::do_call::h6f5c70e8b0a34f92
2022-09-02T12:36:53.9061543Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:492:40
2022-09-02T12:36:53.9062149Z   25:     0x55987e5f6d6e - std::panicking::try::h68766ba264ecf2e2
2022-09-02T12:36:53.9062659Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:456:19
2022-09-02T12:36:53.9063285Z   26:     0x55987e5f6d6e - std::panic::catch_unwind::hc36033d2f9cc04af
2022-09-02T12:36:53.9063798Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panic.rs:137:14
2022-09-02T12:36:53.9064465Z   27:     0x55987e5f6d6e - std::rt::lang_start_internal::{{closure}}::h78c037f4a1a28ded
2022-09-02T12:36:53.9064983Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/rt.rs:128:48
2022-09-02T12:36:53.9065609Z   28:     0x55987e5f6d6e - std::panicking::try::do_call::he6e1fffda4c750ee
2022-09-02T12:36:53.9066137Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:492:40
2022-09-02T12:36:53.9066750Z   29:     0x55987e5f6d6e - std::panicking::try::h48a77ddbb2f4c87a
2022-09-02T12:36:53.9067263Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panicking.rs:456:19
2022-09-02T12:36:53.9067971Z   30:     0x55987e5f6d6e - std::panic::catch_unwind::hfa809b06a550a9e7
2022-09-02T12:36:53.9068481Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/panic.rs:137:14
2022-09-02T12:36:53.9069116Z   31:     0x55987e5f6d6e - std::rt::lang_start_internal::h4db69ed48eaca005
2022-09-02T12:36:53.9069626Z                                at /rustc/4b91a6ea7258a947e59c6522cd5898e7c0a6a88f/library/std/src/rt.rs:128:20
2022-09-02T12:36:53.9070070Z   32:     0x55987e697432 - main
2022-09-02T12:36:53.9070471Z   33:     0x7f2a7b911083 - __libc_start_main
2022-09-02T12:36:53.9070859Z   34:     0x55987e59c5e9 - _start
2022-09-02T12:36:53.9071230Z   35:                0x0 - <unknown>
2022-09-02T12:36:53.9084227Z [0m[0m[1m[31merror[0m[1m:[0m bench failed
2022-09-02T12:36:53.9118903Z ##[error]Process completed with exit code 101.
```

--- a/cli/bench/main.rs
+++ b/cli/bench/main.rs
@@ -523,6 +523,7 @@ async fn main() -> Result<()> {
         ])
         .args(args.iter())
         .stdout(Stdio::null())
+        .env("LC_NUMERIC", "C")
         .spawn()?
         .wait()?;
       let expected_exit_code = expected_exit_code.unwrap_or(0);

